File: src/Projects/Android/015-TextGeneratorSystem/Server/src/main/java/org/csystem/app/generator/text/server/configuration/datetime/DateTimeConfig.java
package org.csystem.app.generator.text.server.configuration.datetime;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Scope;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

@Configuration
public class DateTimeConfig {
    @Bean
    @Scope("prototype")
    public LocalDateTime createDateTime()
    {
        return LocalDateTime.now();
    }
}

File: src/Projects/Android/ChatSystem/Server/src/main/java/org/csystem/app/chat/server/configuration/datetime/DateTimeConfig.kt
package org.csystem.app.chat.server.configuration.datetime

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Scope
import java.time.LocalDateTime

@Configuration
open class DateTimeConfig {
    @Bean
    @Scope("prototype")
    open fun createDateTime(): LocalDateTime
    {
        return LocalDateTime.now()
    }
}
